wazo_sysconfd/settings.py

```diff
@@ -0,0 +1,12 @@
+from functools import lru_cache
+
+from wazo_sysconfd.main import config
+from wazo_sysconfd.request_handlers.request import RequestHandlersProxy
+
+
+@lru_cache()
+def get_request_handlers_proxy():
```

Comment:
Why does this proxy go in the core `settings.py`, when it is only required in the plugin `request_handlers`? IMHO it should be part of the `Plugin`, so that it can access the `config` without making it a global variable.

Reply:
settings.py removed

the function has moved into the request_handlers plugin.

the config is no more 'global'